packages/interop-tests/src/__test__/interop.test.ts

```diff
@@ -62,20 +62,21 @@ let serverDestination: Destination;
 let browserAddress: Address;
 let browserDestination: Destination;
 
-beforeAll(() => {
+beforeAll(async () => {
   provider = new providers.JsonRpcProvider(rpcEndpoint);
   assetHolderContract = new Contract(
     ethAssetHolderAddress,
     ContractArtifacts.EthAssetHolderArtifact.abi,
     provider
   );
   mineOnEvent(assetHolderContract);
+  // TODO: The onSendMessage listener can still be executing
+  // so we can't properly restart the engine
```

Comment:
What makes it so that `onSendMessage` can still be executing? Should it be, after a test case is finished? It seems to me like a test case is _not finished_, if the browser is still sending a message to the server.

Reply:
I think this [comment](https://github.com/statechannels/statechannels/pull/3546#discussion_r641679623) covers it